README.md

```diff
@@ -1,30 +1,13 @@
+<!-- Issue templates and Contributing guides are taken from the fastlane repository and adjusted for our needs (https://github.com/fastlane/fastlane/tree/master/.github/ISSUE_TEMPLATE) and  -->
```

Comment:
The `and` at the end of the sentence looks like a stray/artifact. Or got something lost while copy-editing?
```suggestion
<!-- Issue templates and Contributing guides are taken from the fastlane repository (https://github.com/fastlane/fastlane/tree/master/.github/ISSUE_TEMPLATE) and adjusted for our needs  -->
```
From a readability perspective, I think would be better to place the URL directly to where the repository is mentioned - but that is personal preference.